system-int-test/test/cmr/system_int_test/search/collection_directory_pages_test.clj

```diff
@@ -21,37 +24,74 @@
 ;;; Constants and general utility functions for the tests
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(def ^:private base-url
-  "We don't call to `(transmit-config/application-public-root-url)`
-   due to the fact that it requires a context and we're not creating
-   contexts for these integration tests, we're simply using an HTTP
-   client."
-   (format "%s://%s:%s/"
-           (transmit-config/search-protocol)
-           (transmit-config/search-host)
-           (transmit-config/search-port)))
+(def ^:private test-collections (atom {}))
+
+(defn- add-token-header
+  ([token]
+   (add-token-header token {}))
+  ([token options]
+   (assoc-in options [:headers transmit-config/token-header] token)))
 
 (defn- get-response
-  [url-path]
-  (->> url-path
-       (str base-url)
-       (client/get)))
+  ([url-path]
+   (get-response url-path {}))
+  ([url-path options]
+   (-> (transmit-config/application-public-root-url :search)
+       (str url-path)
+       (client/get options))))
+
+(defn- do-permission-assertions
```

Comment:
I'd move this down to right above the test that uses it and add a docstring.

Reply:
Addressed with the move below.